### PR TITLE
Stories Hendler

### PR DESCRIPTION
**Pull request by EightShift**
Added Stories Hendler for beautiful align stories images with blurred substrate for images with NOT 9:16 aspect ratio.
Replacing the photo handler and remove compatible_aspect_ratio() in upload_story_photo().

Stories rendering like:
milkyway jpeg STORIES

### CODE REVIEW

**Comment by ohld** [comment]
Thanks for your new methods. Please all codestyle and CI issues, and I'll merge

**Comment by EightShift** [comment]
@ohld, i think it's done.

**Comment by ohld** [comment]
Thanks!
Since you are adding new functionality and methods to the bot, may I ask you to add docstrings to simplify the understanding of new methods for old and new users? Thanks

**Comment by maxvinters** [comment]
Sweet feature, bro)

**Comment by steffanjensen** [comment]
Did you see some of the user_tags features inside stories params? Have been trying to implement that for a week.

**Comment by ohld** [comment]
@EightShift Can you please add docstrings for functions and methods to clarify your parameters and methods? Thanks. I'll merge then.

**Comment by EightShift** [comment]
@ohld Like this?
It's my first pull request... 😖

File: instabot/api/api_photo.py [hunk 1]
```diff
@@ -246,3 +246,70 @@ def resize_image(fname):
...
+    img = Image.open(fname)
+    if (img.size[0], img.size[1]) == (1080, 1920):
+        print("Image is already 1080x1920. Just converting image.")
+        new_fname = "{}.STORIES.jpg".format(fname)
+        new = Image.new("RGB", (img.size[0], img.size[1]), (255, 255, 255))
+        new.paste(img, (0, 0, img.size[0], img.size[1]))
+        new.save(new_fname)
+        return new_fname
+    else:
+        min_width = 1080
```
Comment (ohld) [inline instabot/api/api_photo.py:276]: looks like these are the constants. I'd suggest making them global variables at the beginning of the file. And replace all `1080` and `1920`.

**Review by ohld — APPROVED** [review]

**Comment by ohld** [comment]
I can't be sure that your code is well written and true-pythonic, but your new functionality is great! I'll merge it with pleasure. Can't wait for more fancy features from your side!

**Comment by EightShift** [comment]
@ohld > I can't be sure that your code is well written and true-pythonic, but your new functionality is great! I'll merge it with pleasure. Can't wait for more fancy features from your side!

I agree. I'm a newbie in Python...
Maybe I can make the code better. 😅

I am glad that I can add new features to the instabot, because I started to learn the python through instabot.

**Pull request merged**